internal/proxy/task_insert.go

```diff
@@ -183,7 +183,7 @@ func (it *insertTask) PreExecute(ctx context.Context) error {
 	var rowIDBegin UniqueID
 	var rowIDEnd UniqueID
 	tr := timerecord.NewTimeRecorder("applyPK")
-	rowIDBegin, rowIDEnd, _ = it.idAllocator.Alloc(rowNums)
+	rowIDBegin, rowIDEnd, _ = AllocAutoID(it.idAllocator, rowNums)
```

Comment:
are there other places where we need to allocate auto id? such as bulk import? we need to make sure we do this everywhere